cmd/list/rhRegion/cmd.go

```diff
@@ -0,0 +1,71 @@
+package rhRegion
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/config"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var (
+	args struct {
+		discoveryURL string
+	}
+)
+
+var Cmd = NewRhRegionCommand()
+
+func NewRhRegionCommand() *cobra.Command {
+	Cmd := &cobra.Command{
+		Use:   "rh-regions",
+		Short: "List available OCM regions",
+		Long:  "List available OCM regions",
+		Example: `  # List all supported OCM regions 
+ocm list rh-regions`,
+		Run:    run,
+		Hidden: true,
+	}
+
+	flags := Cmd.Flags()
+	flags.StringVar(
+		&args.discoveryURL,
+		"discovery-url",
+		"",
+		"URL of the OCM API gateway. If not provided, will reuse the URL from the configuration "+
+			"file or "+sdk.DefaultURL+" as a last resort. The value should be a complete URL "+
+			"or a valid URL alias: "+strings.Join(ocm.ValidOCMUrlAliases(), ", "),
+	)
+	return Cmd
+}
+
+func run(cmd *cobra.Command, argv []string) {
+	r := rosa.NewRuntime()
+	defer r.Cleanup()
+	cfg, _ := config.Load()
+	gatewayURL, err := ocm.ResolveGatewayURL(args.discoveryURL, cfg)
+	if err != nil {
+		r.Reporter.Errorf("Failed to determine gateway URL: %v", err)
+		os.Exit(1)
+	}
+
+	fmt.Fprintf(os.Stdout, "Discovery URL: %s\n\n", gatewayURL)
```

Comment:
is there a reason to use fmt.Fprintf over using the reporter?

Reply:
Ideally yes, unless it requires some specific formatting not in the reporter

Reply:
Added writer